### PR TITLE
Some rephrasing

### PR DESCRIPTION
**Pull request by casparml**

### CODE REVIEW

File: src/content/docs/settings/linking.mdx
Comment (Luciism) [inline src/content/docs/settings/linking.mdx:1]: "By linking a player to your Discord account, you can use commands without having to specify their username every time." doesn't make as much sense

Comment (casparml) [reply src/content/docs/settings/linking.mdx:1]: how?

Comment (Luciism) [reply src/content/docs/settings/linking.mdx:1]: "their" infers that the minecraft account is not owned by the user

Comment (casparml) [reply src/content/docs/settings/linking.mdx:1]: "By linking a player to your Discord account, you can use commands without having to specify the username every time." Better?

Comment (Luciism) [reply src/content/docs/settings/linking.mdx:1]: isnt that what i did?

Comment (casparml) [reply src/content/docs/settings/linking.mdx:1]: This is you: "Linking a player to your Discord account will allow you to use commands without specifying the player everytime."

this what we agree on? "By linking a player to your Discord account, you can use commands without having to specify the username every time."

Comment (Luciism) [reply src/content/docs/settings/linking.mdx:1]: okay

**Pull request merged**